components/support/sync-telemetry/metrics.yaml

```diff
@@ -134,6 +134,132 @@ history_sync:
     expires: never
     lifetime: ping
 
+passwords_sync:
```

Comment:
In mozilla/glean_parser#146 we'll be restricting names to disallow underscores. Please use dashes (`-`) instead of underscores (`_`).

Reply:
Will do, thanks! @Dexterp37 should I rename other pings, or leave them as-is?

Reply:
Please leave the other as they are, we added exceptions for them.

Reply:
:+1: